dbmanager/dbmanager.go

```diff
@@ -0,0 +1,93 @@
+package dbmanager
+
+import (
+	"database/sql"
+	"errors"
+	"fmt"
+
+	_ "github.com/lib/pq" //db driver for postgres
+	"golang.org/x/crypto/bcrypt"
+)
+
+var ErrUserNonexistant = errors.New("User does not exist")
+
+type DBManager struct {
+	dbUser, dbPass, dbName string
+	DB                     *sql.DB
+}
+
+//New creates a new DBManager instance and returns it
+func New(user, pass, name string) (DBManager, error) {
+	d := DBManager{dbUser: user, dbPass: pass, dbName: name}
+	err := d.connectToPSQL()
+	return d, err
+}
+
+func (db *DBManager) getPasswordHash(email string) ([]byte, error) {
+	result := db.DB.QueryRow(`SELECT password from users WHERE email = $1`, email)
+
+	var hash []byte
+	if err := result.Scan(&hash); err != nil {
+		return nil, err
+	}
+	if len(hash) != 60 {
```

Comment:
What's this for?

Reply:
You're right, can remove since the hash will be invalidated in the `CheckUserCredentials` function